lib/template-bundler.js

```diff
@@ -15,7 +15,7 @@ export function bundleTemplate(pattern, fileName, _opts) {
 
   jspm.setPackagePath(options.packagePath);
 
-  var builder = new jspm.Builder();
+  var builder = new jspm.Builder(options.builder);
```

Comment:
@Mordred  could you show how you would write the configuration for that? I mean the structure and format of `options.builder`